### PR TITLE
KBC-1884 improve tests for list events

### PR DESCRIPTION
**Pull request by martinjunger**
Jira: KBC-1884

Before asking for review make sure that:

- [x] New client method(s) has tests
- [x] Apiary file is updated
- [x] You declared if there is a BC break or not (will affect next release of a client)

---

### CODE REVIEW

File: tests/Backend/CommonPart1/BucketsTest.php [hunk 1]
```diff
@@ -113,8 +115,26 @@ public function testBucketDetail()
 
     public function testBucketEvents()
     {
-        $events = $this->_client->listBucketEvents($this->getTestBucketId());
-        $this->assertNotEmpty($events);
+        // create unique bucket name to be sure that bucket events are for this bucket
+        $bucketName = sprintf('BucketsTest_testBucketEvents-%s-%s', time(), rand(1000, 9999));
```
Comment (fejjo) [inline tests/Backend/CommonPart1/BucketsTest.php:119]: tie buckety sa tam budu hromadit pretoze pri dalsom behu sa vytvori novy, mozes tam pouzit nieco taketo nie ?
```
$description = 'testBycketEvents';
$this->initEmptyBucket($this->getTestBucketName($description), $stage, $description);
```
a bude to uz aj paralel test ready

**Review by fejjo — CHANGES_REQUESTED** [review]

File: tests/Backend/CommonPart1/BucketsTest.php (hunk 1 above)
Comment (martinjunger) [reply tests/Backend/CommonPart1/BucketsTest.php]: upraveno, dik

File: tests/Common/BranchEventsTest.php [hunk 2]
```diff
@@ -114,7 +115,7 @@ public function testCreateExternalEvent()
         ]);
         $this->assertCount(0, $clientEventList);
 
-        $this->assertTrue(count($this->_client->listEvents()) > 1);
+        $this->assertGreaterThan(1, count($this->_client->listEvents()));
```
Comment (fejjo) [inline tests/Common/BranchEventsTest.php:118]: ❤️

**Review by fejjo — APPROVED** [review]
🙆‍♂️

**Pull request merged**